spec/factory_bot/internal_spec.rb

```diff
@@ -8,6 +8,11 @@
         from(0).
         to(1)
     end
+
+    it "returns the registered trait" do
```

Comment:
Is it important that it returns the registered trait? Would it matter if we eventually return something else? If not, we might not need this test.

Reply:
```register_trait``` and ```register_sequence``` both have the same signature in the method, I thought it was necessary, that's why I added the test, I only see it used in ```FactoryBot::Syntax::Default::DSL``` and the result of using the block in the DSL is not documented(to the best of my knowledge) to use the return value. Maybe we are safe removing the return statement? There is a test under ```attributes/dynamic_spec``` that uses the return value.